src/css/bloblang-playground.css

```diff
@@ -0,0 +1,187 @@
+.bloblang-playground .doc {
+  max-width: unset;
+  margin-right: 48px;
+}
+
+/* Main container setup */
+.bloblang-playground .playground-container {
+  display: flex;
+  font-family: var(--body-font-family);
+  gap: 20px;
+  flex-wrap: wrap;
+}
+
+/* Editor section (left side) */
+.bloblang-playground .editors {
+  flex: 1 1 calc(50% - 20px);
+  display: flex;
+  flex-direction: column;
+  color: var(--code-font-color);
+  box-sizing: border-box;
+  gap: 10px;
+}
+
+.bloblang-playground .doc .choices {
+  flex: 0.4 !important;
+  z-index: 2;
+}
+
+.bloblang-playground .doc .choices[data-type*=select-one] .choices__input {
+  background: var(--body-background);
+  color: var(--body-font-color);
+}
+
+.button-bar {
```

Comment:
does this considers mobile?

Reply:
Tested on mobile, and it is responsive. The buttons wrap.